### PR TITLE
Move "Device Trust" to a top-level docs item

### PR DESCRIPTION
**Pull request by codingllama**
Move "Device Trust" from "Manage Access"->"Guides" to directly below "Manage Access".

### CODE REVIEW

**Comment by codingllama** [comment]
FYI @zmb3, following your feedback from a while back.

File: docs/config.json [hunk 1]
```diff
@@ -476,6 +468,14 @@
             }
           ]
         },
+        {
+          "title": "Device Trust",
```
Comment (codingllama) [inline docs/config.json:472]: I placed it right below "Login Rules", which is another preview feature. Not sure if we have a set ordering for these, so let me know what you think.

**Review by zmb3 — APPROVED** [review]
Nice! This will make it more prevalent on our docs and also create space for Jamf and Intune guides.

**Review by ptgott — APPROVED** [review]

**Comment by codingllama** [comment]
Yes, right after mailing this I realized I would need the extra section for the guides. :)

I'm wondering if I should follow the more traditional setup of making `/access-controls/device-trust/` a landing page and move the actual guide under `/access-controls/device-trust/guide.mdx`.

WDYT @zmb3 @ptgott ?

**Comment by codingllama** [comment]
I went ahead and did the split in 152674adb24fd2eac42ae972d80c67347f705ebd, let me know what you make of it.

**Comment by codingllama** [comment]
Friendly ping @zmb3 @ptgott? What do you think about the split between landing page and guide?

**Comment by codingllama** [comment]
FYI, https://github.com/gravitational/teleport/pull/28167 follows up from this by adding Jamf-related docs.

**Review by ptgott — APPROVED** [review]

**Pull request merged**